src/neo/SmartContract/Helper.cs

```diff
@@ -13,6 +13,8 @@ namespace Neo.SmartContract
 {
     public static class Helper
     {
+        private const long MaxVerificationGas = 0_50000000;
```

Comment:
`ConsensusPayload` has `0_02000000` maybe we should unify these values.

Reply:
`ConsensusPayload` doesn't pay, and it only do the simple signature verification.

Reply:
I think this value should be a part of the policy contract. We may see something on the mainnet that would suggest raising or lowering it, execution cost and GAS price could also change, so we're better be adaptive. And this value is a policy in its essence.

Reply:
I don't think it will be changed often.

Reply:
It's not a bad idea

Reply:
I prefer to just move this line in policy contract, and it doesn't need to be configured.

Reply:
If we're to have it in the policy contract (as a proper policy value, of course), any changes to it are going to be *transparent* and clearly visible on-chain, not like some new node version magically doing something behind the scenes. And changing it in any way won't require releasing a new node version (and then dealing with nodes treating transactions in a different way because of the difference in this limit).

Reply:
> I prefer to just move this line in policy contract, and it doesn't need to be configured.

We can move it to policy, and in the future we can configure it very easy if it's needed

Reply:
I think so as well,better to be in Policy.

Reply:
I don't think so. Even this value can be different for each node, it will not have much impact. Because it only effects verifications.

Reply:
I see,Erik.
But are there bad aspects in moving it to Policy? In summary,I imagine that it has been good to have been moving parameters to the Native Policy.

Reply:
We can't put everything into policy.

Reply:
If we can run large scale applications perhaps we can include many things on the Blockchain Governance Smart Contract, Erik.
I understand that everything may be too much...aheuaheuaea But I still think it is a good practice to move parameters towards there.